lib/handlers/allow.js

```diff
@@ -27,9 +27,11 @@ function allow (mode) {
       suffix: ldp.suffixAcl,
       strictOrigin: ldp.strictOrigin
     })
+    req.acl = acl
 
     getUserId(req, function (err, userId) {
       if (err) return next(err)
+      req.userId = userId
```

Comment:
I don't think you need this part; the userId is already recorded on the request object upstream of this handler.